### PR TITLE
Body map frame

### PR DESCRIPTION
**Pull request by sittr7**
Features:
-2 columns (shown with background color that can be removed later)
-temporary body map graphic from internet
-question + checkbox labels from sample_app

render_body_map.js -- includes all features listed above, some links to other data files may need to be changed later (currently the old path to certain files).

test_body_map.html -- shows body map template, minimal design (only frame div)

Note**
1. It shows some changes I made to intro layout template but it's still a working process. Need to make multiple pages + navigation between pages to be complete.
2. While renaming files for intro layout and body_map layout, some files were duplicated and need to be deleted. I decided to delete the duplicate files first before pulling req (had to close + reopen this pull req).

### CODE REVIEW

File: appside/render_intro_frame.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        graphic.setAttribute("height", "228");
+        frame.appendChild(graphic);
+    }
+
+    let old_frame = $('#frame')[0];
+    old_frame.replaceWith(frame);
+
+ }
+
+/**
```
Comment (lrperlmu) [inline appside/render_intro_frame.js]: delete this code now that it's no longer used.

File: appside/render_body_map.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    let right = document.createElement('div');
+    right.style.backgroundColor = "lightblue";
+    right.style.left = '300px';
+    right.style.height = '100%';
+    right.style.position = 'absolute';
+
+
+	// body maps graphic column
+    var graphic = document.createElement('img');
+    graphic.setAttribute("src", "https://upload.wikimedia.org/wikipedia/en/0/0e/Outline-body.png");
```
[the remark this answers is not in the document]
Comment (sittr7) [reply appside/render_body_map.js]: Edited sample_app.js

File: appside/render_body_map.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    	let name = 'label' + i;
+    	i++;
+
+    	let check = document.createElement('input');
+    	$(check).attr('type', 'checkbox');
+    	$(check).attr('id', name);
+    	right.appendChild(check);
+
+    	let label = document.createElement('label');
+    	$(label).text(stmt);
```
[the remark this answers is not in the document]
Comment (sittr7) [reply appside/render_body_map.js]: Added

File: appside/render_intro_frame.js (hunk 1 above)
Comment (sittr7) [reply appside/render_intro_frame.js]: Deleted.

File: appside/render_body_map.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    render_bodymap_frame(frame);
+}
+
+/**
+
+ * Render a frame for the body maps template.
+ *
+ * @param frame -- Object containing the frame's data. Expected fields:
+ *      frame.title (string)
+ *		frame.question (string) -- text before checkboxes
```
[the remark this answers is not in the document]
Comment (sittr7) [reply appside/render_body_map.js]: Will attempt to change its setting + manually fix all existing files.

**Comment by sittr7** [comment]
I've fixed each requested change on top of the styling suggestions (indents, "", /n) but otherwise left styling unchanged for now. Will definitely add JS Style Guide to my to-do list though.
Let me know if there's anything else specific you wanted me to change :)

**Comment by lrperlmu** [comment]
ship it! 🚢

**Comment by lrperlmu** [comment]
looks like this hasn't been merged yet.

**Comment by lrperlmu** [comment]
This branch can't merge in cleanly. Looks like there are merge conflicts in sample_app.js. This is no big deal, but will take some manual work. Let's do it together to make sure that both of our edits to sample_app.js get preserved.

**Pull request merged**